src/containers/flows/SetBuyNow/SetBuyNow.tsx

```diff
@@ -48,15 +48,22 @@ const SetBuyNow = ({
 	account,
 }: SetBuyNowProps) => {
 	const editText = account !== domain?.owner ? 'selecting' : 'purchasing';
+	const getTitle = () =>
+		isLoadingDomainData
+			? ''
+			: domain?.currentBuyNowPrice?.gt(0)
+			? 'Edit Buy Now'
+			: 'Set Buy Now';
```

Comment:
nit: This one is a one liner so probably doesn't need to be a function. Could just do `wizardHeader = ...` even though it's a bit of a mouthful.

Reply:
Cheers mate yeah thats a sleep mistake 😆 noted to rework in one of next PRs. thanks!